frontend/Controls/+mobile/CurrencySquare.qml

```diff
@@ -0,0 +1,91 @@
+import QtQuick 2.7
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+import QtQuick.Window 2.2
+
+Rectangle {
+    property url currencyType: '../icons/BTC-color.svg'
+    property string currencyType2: "BTC"
+    property string percentChange: "+%.8"
+    property string amountSize: "22.54332 BTC"
+    property string totalValue: "$43,443.94"
+    property string value: "$9,839.99"
+
+    id: square
+    color: "#42454F"
+    width: Screen.width - 55
+    height: 75
+    radius: 8
+
+    Image {
+        id: icon
+        // source: '../icons/BTC-color.svg'
```

Comment:
Can this and the rest of the commented lines in this file be removed?

Reply:
preferably not, it's for reference as I build out